plugin/wakatime.vim

```diff
@@ -6,7 +6,7 @@
 " Website:     https://wakatime.com/
 " ============================================================================
 
-let s:VERSION = '8.0.0'
+let s:VERSION = '8.0.1'
```

Comment:
Let's bump versions in their own commits, so we can update the changelog at the same time.

Reply:
Just for next time, let's merge this as-is this time.